root/etc/confd/templates/yaml.ini.tmpl

```diff
@@ -0,0 +1,27 @@
+; Enable YAML extension module
+{{ $enabled := getenv "PHP_YAML" }}
+{{ if eq $enabled "true" }}
```

Comment:
Is there any variable pollution we need to worry about here? I just noticed that the other templates (using xhprof and xdebug as the example) use what appears to be a namespaced variable which would indicate this one should perhaps be $phpyaml. Other than that (which might not even be relevant) this one looks fine to me.

Reply:
There is not any name pollution. I used enabled here instead of something like $phpyaml because I felt it was more clear. It is my thought to change the others to match, but as there's no functional impact I didn't expand the scope of this change to cover that. Should I?

Reply:
I don't think there's any need to expand the scope here to cover adjusting the others to promote consistency. Thanks for the info on the scoping as I lose track of the rules on all the different systems.